webservices/resources/filings.py

```diff
@@ -0,0 +1,67 @@
+from flask.ext.restful import Resource
+
+from webservices import args
+from webservices import docs
+from webservices import spec
+from webservices import utils
+from webservices import schemas
+from webservices.common import models
+from webservices.common.util import filter_query
+
+fields = {
+    'committee_id',
+    'beginning_image_number',
+    'report_type',
+    'report_year',
+    'recipt_date',
+    'form_type',
+    'primary_general_indicator',
+    'amendment_indicator',
+}
+
+@spec.doc(
+    tags=['filings'],
+    description=docs.FILINGS,
+    path_params=[
+        {
+            'name': 'committee_id',
+            'in': 'path',
+            'description': docs.COMMITTEE_ID,
+            'type': 'string',
+        },
+    ],
+)
+class FilingsView(Resource):
+
+    @args.register_kwargs(args.paging)
+    @args.register_kwargs(args.make_sort_args(default=['-receipt_date']))
+    @schemas.marshal_with(schemas.FilingsPageSchema())
+    def get(self, committee_id=None, **kwargs):
+        filings = models.Filings.query
+        if hasattr(filings, 'committee'):
```

Comment:
Ok, I thought we were going to have to nest it for something- if we are not using that at all let's get rid of it.